simpleMail.py

```diff
@@ -67,11 +67,15 @@ def send_email(frm, to, subject, message, attachments=None, maxsize=100000, html
         mimetype = mimetypes.guess_type(attachment)[0]
         _maintype, _subtype = mimetype.split('/')
         att = MIMEBase(_maintype, _subtype)
-        att.set_payload(open(attachment, 'rb').read())
-        encoders.encode_base64(att)
-        file_name=os.path.basename(attachment)
-        att.add_header('Content-Disposition', 'attachment', filename=file_name)
-        msg.attach(att)
+        try:
+            att.set_payload(open(attachment, 'rb').read())
+        except OSError as err:
+            print('OS Error: {}'.format(err))
```

Comment:
What about sending the exception into the email as text? It seems like this might be nice if you want to ensure you get the email. Instead of an invalid attachment you just get some text.

    except OSError as err:
        att = MIMEText('OS Error: {}'.format(err))
        msg.attach(att)